Update application-load-balancer.md (#1525)
SumoLogic/sumologic-documentation: docs/integrations/amazon-aws/application-load-balancer.md

### Collecting Metrics

* Sumo Logic supports collecting metrics using two source types
	* Configure an [AWS Kinesis Firehose for Metrics Source](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source) (Recommended) 
		Or
	* Configure an [Amazon CloudWatch Source for Metrics](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics)
:::note
Namespace for AWS Application Load Balancer Service is AWS/ApplicationELB
:::

> 	* **Metadata**. Add an **account** field to the source and assign it a value which is a friendly name / alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. Metrics can be queried through the “account field”.

Reviewer: @nmohantyctr Please run a `yarn start` to confirm the alignment looks ok

### Collecting Metrics

> Sumo Logic supports collecting metrics using two source types

Reviewer 2: ```suggestion
Sumo Logic supports collecting metrics using two source types:
```

9. For **Source Category**, enter any string to tag the output collected from this Source (category metadata is stored in a searchable field called `_sourceCategory`).

>     * Example sourceCategory: **aws/observability/alb/logs**

Reviewer 2: ```suggestion
    * Example sourceCategory: `aws/observability/alb/logs`
```

10. **Fields**. Click the **+Add Field** link to add custom log metadata [Fields](/docs/manage/fields.md). Add the following **Fields** in the source:

>     * Add an **account** field and assign it a value which is a friendly name / alias to your AWS account from which you are collecting logs. This name will appear in the Sumo Logic Explorer View. Logs can be queried through the “account field”.

Reviewer 2: ```suggestion
    * Add an **account** field and assign it a value that is a friendly name/alias to your AWS account from which you are collecting logs. This name will appear in the Sumo Logic Explorer View. Logs can be queried using the **account** field”.
```

### Threat Intel

The **AWS Application Load Balancer - Threat Intel** dashboard provides insights into incoming requests from malicious sources determined through [Sumo Logic’s Threat Intel feature](/docs/integrations/security-threat-detection/threat-intel-quick-analysis#03_Threat-Intel-FAQ). Panels show detailed information on malicious IPs and the malicious confidence of each threat.

Use this dashboard to:

> * Identify known malicious IPs that are access your load-balancers and use firewall access control lists to prevent them from sending you traffic going forward

Reviewer 2: ```suggestion
* Identify known malicious IPs that are accessing your load-balancers and use firewall access control lists to prevent them from sending you traffic going forward
```
